templates/2_create_tma_mibi_run.ipynb

```diff
@@ -30,14 +30,14 @@
     "This notebook automatically checks the names assigned to the cores on a TMA. In order to get the most benefit out of the notebook, make sure that you've named your FOVs appropriately. The expected format is RNCM, where N is the row and M is the column of the TMA. For example, a core on the third row and second column would be R3C2, and one on the 7th row and first column would be R7C1.  \n",
```

Comment:
<p>Looks like some, but not all, of the references to <code>tma_name</code> were changed to <code>tma_prefix</code> instead. Please make sure everything in here is consistent  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>